packages/runtime/src/useCases/transport/files/UploadOwnFile.ts

```diff
@@ -53,7 +53,7 @@ class Validator extends SchemaValidator<UploadOwnFileValidatableRequest> {
             );
         }
 
-        if (DateTime.fromISO(input.expiresAt) <= DateTime.utc()) {
+        if (input.expiresAt && DateTime.fromISO(input.expiresAt) <= DateTime.utc()) {
```

Comment:
Could you use our `CoreDate` library to perform the datetime arithmetic? It's already a necessary dependency. Not really a big deal, but it's smells funny to me to use two different datetime libraries within a few lines of code :-)

```ts
input.expiresAt && CoreDate.from(input.expiresAt).isSameOrBefore(new CoreDate())
```